src/aggregation.c

```diff
@@ -198,10 +198,10 @@ static int agg_instance_create_name(agg_instance_t *inst, /* {{{ */
       sstrncpy(inst->ident.plugin_instance, AGG_FUNC_PLACEHOLDER,
                sizeof(inst->ident.plugin_instance));
     else if (strcmp("", tmp_plugin) != 0)
-      snprintf(inst->ident.plugin_instance, sizeof(inst->ident.plugin_instance),
+      snprintf(inst->ident.plugin_instance, sizeof(inst->ident.plugin_instance)  + sizeof(AGG_FUNC_PLACEHOLDER),
```

Comment:
If I'm correct, this introduces buffer overflow and has no chances to be merged.

The same is for other `sizeof + sizeof`.

Reply:
Thanks.  I'm going to close this PR since there's a better solution already proposed.